### ISSUE TITLE
Can you upgrade the A2DP profile to the newest?

### ISSUE THREAD

**Issue by orklann** [airander/A2DP-SINK#2]
Hi, can you upgrade the A2DP profile to the newest? It seems the version implemented in this project is an olde one.

**Comment by airander** [comment]
Possibly, is there a benefit of upgrading to the latest?

**Comment by orklann** [comment]
Yep, since some new Android or iOS just implement a new A2DP Source version, so I think implementing a new A2DP Sink version is a good choice.
